getting_started/step_by_step/your_first_game.rst

```diff
@@ -233,9 +235,11 @@ need to do the following:
 First, we need to check for input - is the player pressing a key? For
 this game, we have 4 direction inputs to check. Input actions are defined
 in the Project Settings under "Input Map". Here, you can define custom events and
-assign different keys, mouse events, or other inputs to them. For this demo,
-we will use the default events that are assigned to the arrow keys on the
-keyboard.
+assign different keys, mouse events, or other inputs to them.
+You can define your own custom "move_right" etc input events here, and
+define inputs such as WASD, arrow keys, and/or controller sticks.
+Alternatively, if you want, you can just use the default events called
+"ui_right" etc that are assigned to the arrow keys on the keyboard.
```

Comment:
This seems like overexplaining for this game. Let's get them working with default input actions before talking about custom ones. This paragraph just implies you can do it but then leaves the_reader still wondering how.

Using the default actions (and these should be consistently referred to as "input actions") is what we want them doing, not an alternative choice.